tests/foundry/client/data/documents/journal-entry-page.test-d.ts

```diff
@@ -0,0 +1,17 @@
+import { expectTypeOf } from "vitest";
+
+// @ts-expect-error data is required
+new JournalEntryPage();
+
+const myJournalEntryPage = new JournalEntryPage({});
```

Comment:
I think Luke's most recent changes make this bit in particular not run correctly but it should be
```ts
// @ts-expect-error data is required
new JournalEntryPage();
// @ts-expect-error name is required
new JournalEntryPage({});

const myJournalEntryPage = new JournalEntryPage({ name: 'foo' });
```

Reply:
Thank you for the feedback, I see what I missed :)
This has been fixed :)